src/picongpu/include/simulation_defines/param/particleDetector.param

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright 2016 Heiko Burau
+ *
+ * This file is part of PIConGPU.
+ *
+ * PIConGPU is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * PIConGPU is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PIConGPU.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+namespace picongpu
+{
+namespace particleDetector
+{
+
+/** Map theta and phi into [0,1] respectively. These ranges correspond to
+ * the normalized histogram range of the detector (0: first bin, 1: last bin).
+ * Out-of-range values are mapped to the first or the last bin.
+ *
+ * Useful for tuning the spacial detector resolution.
+ *
+ * \param theta -maxTheta...maxTheta
+ * \param phi -maxPhi...maxPhi
+ * \param maxTheta maximum value of angle theta
+ * \param maxPhi maximum value of angle phi
+ * \return Two values within [-1,1]
+ */
+HDINLINE float2_X mapThetaPhiToNormedRange(const float_X theta,
+                                           const float_X phi,
```

Comment:
the number of bins is a coarse tool for tuning the resolution. If the user needs for example a finer resolution in the middle of the calorimeter, he can change the mapping in the param file here.